ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -19217,17 +19217,6 @@ periodics:
     job-release: "4.6"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-release-master-nightly-4.6-e2e-metal-assisted
-  reporter_config:
-    slack:
-      channel: '#assisted-deployment-ci'
-      job_states_to_report:
-      - failure
-      - error
-      report_template: '{{if eq .Status.State "success"}} :green_jenkins_circle: Job
-        *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs>
-        :green_jenkins_circle: {{else}} :red_jenkins_circle: Job *{{.Spec.Job}}* ended
-        with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :red_jenkins_circle:
-        {{end}}'
```

Comment:
Will this unrelated change go away with a rebase?

Reply:
yes, hopefully. I don't know why it was removed when doing `make jobs`...

Reply:
yeah it was cleaned when i re-ran the command.